onnxscript/function_libs/torch_lib/registration.py

```diff
@@ -40,6 +40,13 @@ def register(self, func: Any, name: str, *, private: bool = False) -> None:
     def __getitem__(self, name):
         return self._registry[name]
 
+    def get_functions(self, name: str):
+        return self._registry.get(name, None)
+
+    def is_registered(self, name: str):
```

Comment:
I think this is implemented in __contains__ below already. One can use

`name in registry` to check

Reply:
No worry. After talk to @BowenBao, I will have these registry APIs in converter most likely. This is just a draft that I am debugging while implementing dispatcher. I will have you review on final version if you are available at the time.

Reply:
I will probably have to implement registry/dispatcher class in converter. And this one would only be imported as part of registration source.

Reply:
Sounds like a plan